keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -50,103 +44,86 @@ class RandomResizedCrop(BaseImageAugmentationLayer):
              constant".  Defaults to `0.0`.
         seed: Integer. Used to create a random seed.
     """
-    def __init__(
-        self,
-        resize_factor,
-        crop_size,
-        aspect_ratio_factor=0.0,
-        interpolation="bilinear",
-        fill_mode="reflect",
-        fill_value=0.0,
-        seed=None,
-        **kwargs,
-    ):
-        super().__init__(seed=seed, **kwargs)
-        self.area_factor = preprocessing.parse_factor(resize_factor,
-                                                      param_name="area_factor",
-                                                      seed=seed)
-        self.aspect_ratio_factor = preprocessing.parse_factor(
-            aspect_ratio_factor, param_name="aspect_ratio_factor", seed=seed)
-        if resize_factor == 0.0 and aspect_ratio_factor == 0.0:
-            warnings.warn(
-                "RandomResizedCrop received both `area_factor=0.0` and "
-                "`aspect_ratio_factor=0.0`. As a result, the layer will perform no "
-                "augmentation.")
+    def __init__(self,
+                 target_size,
+                 aspect_ratio_factor=(3. / 4., 4. / 3.),
+                 area_factor=(0.08, 1.0),
+                 interpolation="bilinear",
+                 fill_mode="reflect",
+                 fill_value=0.0,
+                 seed=None,
+                 **kwargs):
+        super(RandomResizedCrop, self).__init__(seed=seed, **kwargs)
```

Comment:
in python3 you can use super().__init__(...) instead of super(RandomResizedCrop, self)

This holds for all cases.  Before everyone did this to be backwards compatible with python2.